modules/custom/az_person/az_person_profile_import/migrations/az_person_profile_import.yml

```
@@ -0,0 +1,101 @@
+id: az_person_profile_import
+label: Profiles Integration
+migration_tags:
+  - Profiles Integration
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: az_person_profile_import_json
+  urls: []
+  item_selector: /
+  ids:
+    netid:
+      type: string
+  constants:
+    NAME: 'name'
+
+  fields:
+    -
+      name: netid
+      selector: 'Person/netid'
+    -
+      name: surname
+      selector: 'Person/surname'
+    -
+      name: givenname
+      selector: 'Person/givenname'
+    -
+      name: email
+      selector: 'Person/email'
+    -
+      name: biography
+      selector: 'Bio/desc'
+    -
+      name: titles
+      selector: 'Titles'
+    -
+      name: phone
+      selector: 'Person/phone'
+    -
+      name: degrees
+      selector: 'Degrees'
+
+process:
+  type:
```

This should be possible now.
```suggestion
  field_az_netid: netid
  type:
```

---

Good idea, I think it would be good to add this.